GOVERNANCE.md

```diff
@@ -74,6 +75,7 @@ The current team members are:
 - Peter Štibraný — [@pstibrany](https://github.com/pstibrany) ([Grafana Labs](https://grafana.com/))
 - Tyler Reid — [@treid314](https://github.com/treid314) ([Grafana Labs](https://grafana.com/))
```

Comment:
Do we want to keep Justin and Tyler since they're no longer on call at GL

Reply:
Updating it too.